packages/expo-modules-core/expo-modules-library/src/main/kotlin/expo/modules/plugin/ExpoModulesCoreLibraryPlugin.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+package expo.modules.plugin
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.internal.extensions.core.extra
+
+private const val defaultKotlinVersion = "1.9.24"
+private const val defaultKSPVersion = "1.9.24-1.0.20"
+
+
+abstract class ExpoModulesLibraryPlugin : Plugin<Project> {
+  override fun apply(project: Project) {
+    if (project.pluginManager.hasPlugin("com.android.application")) {
```

Comment:
```suggestion
    if (!project.pluginManager.hasPlugin("com.android.library")) {
```

maybe this is more correct in case people have other type of subprojects like java project. we don't want to enforce android sdk version for java projects